extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -737,6 +932,19 @@ function constructSRARequest_(win, doc, instances) {
 }
 
 /**
+ * @param {string} member
+ * @param {string} expectedType
+ */
+function verifyRtcConfigMember(member, expectedType) {
```

Comment:
Nit: Appears that you're only verifying boolean types so no need to actually pass it.  Suggest rename to:  logIfRtcConfigMemberNotBoolean(member)